src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Ball {
+    private Color color;
+    private int number;
+
+    public Ball(Color color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return "color: " + color + " & number: " + number;

```

Comment:
The toString method should return a string representation of the object in a more readable format. Consider overriding it to return a string in the format 'Ball{color=..., number=...}'.

Reply:
Done it but in my opinion less means better ;)